qa/workunits/cephadm/test_cephadm.sh

```diff
@@ -370,6 +383,10 @@ $CEPHADM unit --fsid $FSID --name mon.a -- disable
 expect_false $CEPHADM unit --fsid $FSID --name mon.a -- is-enabled
 $CEPHADM unit --fsid $FSID --name mon.a -- enable
 $CEPHADM unit --fsid $FSID --name mon.a -- is-enabled
+$CEPHADM unit --fsid $FSID --name mon.a -- status
+$CEPHADM unit --fsid $FSID --name mon.a -- stop
+expect_return_code 3 $CEPHADM unit --fsid $FSID --name mon.a -- status
```

Comment:
it looks like this isn't passing (see other comment with the logs)

Reply:
I think this is because of `bash -e`, so in the function `expect_return_code`, when the `eval` is called and the command does not return 0, the whole script is terminated.
I surrounded that call to `eval` and the extraction of the return code with a `set +e` and `set -e`, so now it should be good.